android/titanium/src/java/org/appcelerator/titanium/proxy/ColorProxy.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020-Present by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+package org.appcelerator.titanium.proxy;
+
+import org.appcelerator.kroll.KrollProxy;
+import org.appcelerator.kroll.annotations.Kroll;
+
+import android.graphics.Color;
+
+import androidx.annotation.ColorInt;
+
+@Kroll.proxy
+/**
+ * This is a proxy representation of the Android Color type.
+ * Refer to <a href="https://developer.android.com/reference/android/graphics/Color">Android Color</a> for more details.
+ */
+public class ColorProxy extends KrollProxy
+{
+	private Color color;
+
+	public ColorProxy(@ColorInt int colorInt)
+	{
+		this(Color.valueOf(colorInt));
+	}
+
+	public ColorProxy(Color color)
+	{
+		this.color = color;
+	}
+
+	@Kroll.method
+	public String toHex()
+	{
+		if (this.color == null) {
+			return "#000000";
+		}
+		// Padding! If length is less than 8, pad with leading 0s
+		String hex = Integer.toHexString(this.color.toArgb());
+		while (hex.length() < 8) {
+			hex = "0" + hex;
+		}
+		return "#" + hex;
+	}
+
```

Comment:
Perhaps we should add a `toString()` method in case the color object is appended to a string in JavaScript.
```
@Kroll.method
@Override
public String toString()
{
	return toHex();
}
```